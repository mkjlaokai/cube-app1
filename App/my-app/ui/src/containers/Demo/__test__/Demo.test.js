import React from 'react';
import { fromJS } from 'immutable';
import { Demo } from '../index';

describe('Demo Component', () => {
  it('should render correctly', () => {
    const props = {
      t: jest.fn(),
      demo: fromJS({
        count: 0,
      }),
      demoActions: {},
    };
    const wrapper = shallow(<Demo {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
