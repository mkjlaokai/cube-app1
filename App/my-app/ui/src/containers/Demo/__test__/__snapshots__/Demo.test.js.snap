// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo Component should render correctly 1`] = `
<div
  className="demo"
>
  <div
    className="demo-header"
  >
    <img
      alt="logo"
      className="demo-logo"
      src="logo.svg"
    />
    <h2 />
  </div>
  <div
    className="demo-content"
  >
    <div>
      <h3>
        counter:
        0
      </h3>
    </div>
    <div
      className="demo-buttons"
    >
      <Button
        clicked={false}
        ghost={false}
        icon="plus"
        loading={false}
        onClick={[Function]}
        prefixCls="ant-btn"
      />
      <Button
        className="demo-minus-count"
        clicked={false}
        ghost={false}
        icon="minus"
        loading={false}
        onClick={[Function]}
        prefixCls="ant-btn"
      />
    </div>
    <div
      className="demo-buttons"
    >
      <Button
        clicked={false}
        ghost={false}
        loading={false}
        onClick={[Function]}
        prefixCls="ant-btn"
        size="small"
      />
    </div>
  </div>
</div>
`;
